src/components/badge/badge.component.tsx

```diff
@@ -34,11 +37,16 @@ export const Badge = ({
         }
       : {
           "aria-label": ariaLabel,
+          customColor,
```

Comment:
comment: as we're explicitly passing this to `StyledBadge` on line 47 we don't need to conditionally add it to the props here